File: W3Schools/Backend/C/Main_Tutorial/6_Variables/6_Variables.c

// C: W3Schools - Main Tutorial
// Section 6: Variables

 #include <stdio.h>

int main() {
  // Declare a variable
  int myNum;

  // Assign a value to the variable
  myNum = 15;

  // In C, you must also declare the data type of the variable when you assign and print it:
  printf("%d\n", myNum);

  // We can combine variables together, in the case of integers, we can add them together:
  int x1 = 5;
  int y1 = 6;
  int sum = x1 + y1;
  printf("%d\n", sum);

  // We can also declare multiple variables in one line:
  int x2 = 5, y2 = 6, z2 = 50;
  printf("%d\n", x2 + y2 + z2);

  // We can also assign the same value to multiple variables in one line:
  int x3, y3, z3;
  x3 = y3 = z3 = 50;
  printf("%d\n", x3 + y3 + z3); 

  return 0;
}

File: W3Schools/Backend/C#/6_Variables.cs

// C#: W3Schools - Main Tutorial
// Section 6: Variables

// Variables in C# work the same as in any other programming language:
string name = "John";
Console.WriteLine(name);

// Some basic types include:
int myNum1 = 5;
double myDoubleNum = 5.99D;
char myLetter = 'D';
bool myBool = true;
string myText = "Hello";

// A constant is a variable whose value cannot change once it has been assigned:
const int myNum2 = 15;

// NOTE: That constants must be declared with a value.

// Multiple variables can be declared in one line:
int x1 = 5, y1 = 6, z1 = 50;
Console.WriteLine(x1 + y1 + z1);

// Multiple variables can be given the same value:
int x2, y2, z2;    // Declare variables
x2 = y2 = z2 = 50; // Assign the same value to all three variables
Console.WriteLine(x2 + y2 + z2);
